src/main/java/com/williamfeliciano/sba_movies/dtos/MovieDto.java

```diff
@@ -0,0 +1,31 @@
+package com.williamfeliciano.sba_movies.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class MovieDto {
```

Comment:
If you want two dtos to make the difference between a search result endpoint and a get one endpoint, you should add the "Details" suffix to the second dto, as `MovieDetailsDto`.